tests/valid-minimal.yaml

```diff
@@ -23,3 +23,13 @@ telemetry: false
 keyboard: us
 language: en_US.UTF-8
 kernel: kernel-native
+pre-install: [
+   {cmd: 'echo "running pre-install hook form: $chrootDir"'}
```

Comment:
is chrootDir set to "/" when chroot: false ?

Reply:
nope